### PR TITLE
 Prepare for community.aws release 3.0.0

### PR DESCRIPTION
**Pull request by markuman**
##### SUMMARY

next major release

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

everything

### CODE REVIEW

**Comment by markuman** [comment]
recheck

**Comment by markuman** [comment]
recheck

**Comment by markuman** [comment]
recheck

**Comment by markuman** [comment]
recheck

File: galaxy.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace: community
```
Comment (alinabuzachis) [inline galaxy.yml:1]: I guess we also have to update `amazon.aws` version in dependencies to `3.0.0`.

**Review by alinabuzachis — CHANGES_REQUESTED** [review]
I guess you also have to update the amazon.aws version here https://github.com/ansible-collections/community.aws/blob/main/tests/requirements.yml#L2

I guess docs update is missing. Did you generate the docs using [`collection_prep` repo](https://github.com/alinabuzachis/collection_prep)?

**Review by alinabuzachis — APPROVED** [review]

**Comment by markuman** [comment]
recheck

File: meta/runtime.yml [hunk 2]
```diff
@@ -187,23 +187,10 @@ plugin_routing:
   modules:
     ec2_elb:
       redirect: community.aws.elb_instance
```
Comment (tremble) [inline meta/runtime.yml:189]:
```suggestion
```

**Review by tremble — DISMISSED** [review]
Sorry about the delayed response.  Mostly looks good ec2_elb was deprecated and should probably go away.

**Comment by markuman** [comment]
recheck

**Comment by markuman** [comment]
recheck

**Review by alinabuzachis — APPROVED** [review]

**Review by jillr — APPROVED** [review]

**Comment by markuman** [comment]
The gating process is stalling...  
cc @goneri

**Comment by alinabuzachis** [comment]
@markuman I throw it there. Could it be because there seem to be ** Changes requested**? Have you tried to dismiss this?

**Comment by markuman** [comment]
> @markuman I throw it there. Could it be because there seems to be **Changes requested**? Have you tried to dismiss this?

I'll try

**Pull request merged**